docs/tutorials/ort-format-model/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Overview
+parent: Deploy on mobile device and web
+grand_parent: Tutorials
+has_children: false
+nav_order: 1
+---
+{::options toc_levels="2" /}
+
+# Overview
+
+The execution environment on Mobile devices, and for web browsers, have fixed memory and disk storage. The execution scenario on web browsers have strict memory consumption and network bandwidths requirement. Therefore, it is essential that any AI execution library is optimized to consume minimun resources in terms of disk footprint, memory and network usage (both model size and binary size). ONNX Runtime was enhanced to target these size constrained environments. These enhancments are packaged in the ONNX Runtime Mobile and web offering. 
```

Comment:
... library is optimized to consume **minimum** resources in terms of disk ...

...constrained environments. These **enhancements** are ...